lxd/instance/drivers/driver_qemu_templates.go

```diff
@@ -74,6 +74,10 @@ func qemuMachineType(architecture int) string {
 	switch architecture {
 	case osarch.ARCH_64BIT_INTEL_X86:
 		machineType = "q35"
+	case osarch.ARCH_32BIT_ARMV7_LITTLE_ENDIAN:
```

Comment:
```suggestion
	case osarch.ARCH_32BIT_ARMV7_LITTLE_ENDIAN, osarch.ARCH_32BIT_ARMV8_LITTLE_ENDIAN, osarch.ARCH_64BIT_ARMV8_LITTLE_ENDIAN:
```

Reply:
As always, my questions/suggestions on anything related with Go are to be taken with a large grain of salt. @tomponline is really the authoritative voice here. I'm still just a Go noob ;)